### ISSUE TITLE
Error reproducing examples

### ISSUE THREAD

**Issue by tinastefaniewenzel** [airbnb/knowledge-repo#226]
Auto-reviewers: @NiharikaRay @matthewwardrop @earthmancash @danfrankj

I went through the example in the documentation but the example_post.Rmd or example_post.ipynb do not get displayed at all. 

All previous steps work, but when trying: 
`knowledge_repo --repo . preview project/example`

, the app starts at http://127.0.0.1:7000/post/project/example.kp 
and prints out to console: 

```
INFO:knowledge_repo.repositories.gitrepository:Fetching updates to the knowledge repository...
INFO:alembic.runtime.migration:Context impl SQLiteImpl.
INFO:alembic.runtime.migration:Will assume non-transactional DDL.
```

But the browser shows: 

```
Uh oh! Looks like something broke
Try clicking around on the other links while we try and get this page up and running

```

I ran`knowledge_repo --repo . --debug runserver` which returns:
```
INFO:knowledge_repo.repositories.gitrepository:Fetching updates to the knowledge repository...
INFO:alembic.runtime.migration:Context impl SQLiteImpl.
INFO:alembic.runtime.migration:Will assume non-transactional DDL.
INFO:knowledge_repo.repositories.gitrepository:Fetching updates to the knowledge repository...
INFO:alembic.runtime.migration:Context impl SQLiteImpl.
INFO:alembic.runtime.migration:Will assume non-transactional DDL.
WARNI [werkzeug]  * Debugger is active!
```

Any suggestions what the issue may be ?

**Comment by matthewwardrop** [comment]
Hi @tinastefaniewenzel!

Thanks for reporting in!

The currently released version of the Knowledge Repo on PyPI has a bug in the preview command. This has been fixed as of the current master, which will be released as a new version in the near future. In the short term, you can run:

`pip install git+https://github.com/airbnb/knowledge-repo.git#egg=knowledge_repo`

This will install the current master branch onto your machine fixing this bug.

I will close this issue for now, but feel free to reopen it if this doesn't solve your problem.

Thanks again!

**Comment by tinastefaniewenzel** [comment]
👍 It works. Many Thanks!

**Comment by nmusolino** [comment]
@matthewwardrop, thanks for the clear diagnosis and workaround.

I am experiencing an issue with identical symptoms on Windows.  My version of knowledge repo is:
```
In [1]: import knowledge_repo

In [2]: print(knowledge_repo.__version__)
0.7.6_e128d3b0cd895937a9c6f75bd6561976380bde38
```

The [release notes](https://github.com/airbnb/knowledge-repo/releases) for this version mention that preview was fixed, but it is not working for me.  I can confirm that my knowledge repo source code contains the fix for #219.

Do you know if there is any simple way to enable verbose logging, to get flask to log why it's displaying a 404?  E.g. what exception was raised by the routes defined in knowledge repo code.

**Comment by emily-wasserman** [comment]
I am experiencing the same issue currently on Mac (Sierra OS), specifically trying to recreate the Markdown post example. Like @nmusolino I have v.0.7.6. 

For me, the issue is resolved if I change ` ```{python}` to ` ```python` above Python code cells - i.e., I can highlight Python syntax but not run code. However, the R cells run perfectly with ` ```{r}`.  My Python version is 3.6.4, if that helps.
